arm/assets/myoarm_assets.xml

```diff
@@ -0,0 +1,838 @@
+﻿<mujocoinclude model="MyoArm_v0.01">
+<!-- =================================================
+	Copyright 2023 Vikash Kumar, Vittorio Caggiano
+	Model	:: MyoArm (MuJoCoV2.0)
+	Author	:: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano
+	source	:: https://github.com/vikashplus
+	License	:: Under Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+====================================================== -->
+
+    <!-- This model has been converted from an OpenSim model. Model conversion by MyoConverter https://github.com/MyoHub/myoConverter -->
```

Comment:
this is a combinations of models which were not all converted via myoconverter

Reply:
What is the best way to handle this? If you know all the details can I have you update the read me accordingly and I can remove this line from here? 
https://github.com/MyoHub/myo_sim/tree/dev/arm

Reply:
I removed it from here. More details can go to the README